Remotion/Web/Test.Shared/MultiplePostBackCatching/TestControlGenerator.cs

```diff
@@ -69,7 +70,8 @@ public bool IsAlertHyperLink (Control control)
       if (control is HyperLink)
       {
         HyperLink hyperLink = (HyperLink)control;
-        if (hyperLink.NavigateUrl == "#" && hyperLink.Attributes["onclick"].Contains("alert"))
+        var url = SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>().GetURL();
```

Comment:
This is just a helper type used in our tests. we can stick with "#" to keep things simple and isolated.
(revert entire file)

Reply:
Are you sure we dont want to include the Fallback Naviationprovider? There is no other way than the individual control tests for us to see the behavior of the service otherwise.

Reply:
@a-ctor can show you the web-test testsites (if you don't find them) and you need more than the indiv-controls and the tabbed test. I think there's everything you need availble between those. The multi-postback stuff only tests some cases, much less than the other pages.